libs/perun/components/src/lib/attribute-critical-operations-toggles/attribute-critical-operations-toggles.component.html

```diff
@@ -1,19 +1,35 @@
 <mat-slide-toggle
-  class="toggle-font mb-25"
+  [class]="readOperation ? 'toggle-font' : 'toggle-font mb-25'"
```

Comment:
You can enable specific class (perhaps multiple not sure) like so. 
```suggestion
  class="toggle-font"
  [class.mb-25]="!readOperation"
```